setup.cfg

```
@@ -33,7 +33,7 @@ install_requires =
     sdcflows ~= 1.2.0
     smriprep ~= 0.5.2
     tedana >= 0.0.5
-    templateflow >= 0.4.2
+    templateflow ~= 0.5.1rc1
```

esp. with RCs (`~=` will not match 0.5.1), I think we want this:
```suggestion
    templateflow >= 0.5.1rc1
```
Also, templateflow does not generally break it's API - so it boils down to the actual templates in the skeleton. I think it is safe to use a blanket `>=` here.